lib/json_schema_form/umm_preview_elements/umm_preview_service_organizations.rb

```diff
@@ -1,50 +1,51 @@
 # Renders cards for Service Organizations
 
 # :nodoc:
+class UmmPreviewServiceOrganizations < UmmPreviewOrganizations
 # TODO: MMT-2267 Abstract these and tool preview elements to minimize repeated
```

Comment:
I think you did this TODO.

Reply:
I think that there might be more that can be done, I did not spend a ton of time looking at all the parts. I also think there is some overlap with the contacts cards also. I'm inclined to leave it, and you can still take a look in that ticket. If you deem it unnecessary then you don't need to do more.